motoman_driver/launch/motion_streaming_interface.launch

```diff
@@ -3,10 +3,10 @@
 -->
 <launch>
 	<!-- IP of robot (or PC running simulation) -->
-	<arg name="robot_ip" />
+	<arg name="robot_ip" doc="IP of controller (only required if not using industrial simulator)" />
```

Comment:
This launch file cannot be used with the `industrial_robot_simulator`, so this seems like a strange comment as a doc string.

Can you clarify?

Reply:
Additionally: the `arg` is really a required arg (there is no `default` attribute), so it's always required.

Reply:
Somehow overlooked the missing default. Modified.